labgrid/util/qmp.py

```diff
@@ -27,7 +27,7 @@ def _negotiate_capabilities(self):
 
     def _read_parse_json(self):
         line = self.monitor_out.readline().decode('utf-8')
-        self.logger.debug("Received line: %s", line)
+        self.logger.debug("Received line: %r", line)
```

Comment:
```suggestion
        self.logger.debug("Received line: %s", line.rstrip("\r\n"))
```

Reply:
Good point, I've changed it according to your suggestion.